### PR TITLE
Issue #23 - Fix wording and grammar 

### PR DESCRIPTION
**Pull request by treetrunk**
This covers everything mentioned in Issue #23 except for editing the map generator section on the main page. I could not think of a way to word it without making the Map Gen section unreasonably long. This is my first time contributing to open source projects, so if I am doing anything incorrectly, please let me know.

### CODE REVIEW

**Comment by rubenwardy** [comment]
You also add a gallery to the customize page, you should state that :P

File: downloads.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 <p class="intro">
   You may also want to look at some <a href="https://forum.minetest.net/viewforum.php?f=15">subgames</a>. Subgames form a foundation for the game using Lua scripts. Different subgames have different objectives, such as survival, building or Player vs Player.
-  Usually Minetest comes with <a href="https://github.com/minetest/minetest_game">minetest_game</a>, to supply the default items and blocks.
+  Usually Minetest comes with <a href="https://github.com/minetest/minetest_game">minetest_game</a>, to supply the default items and blocks. After you've tried Minetest you might want to try some mods or subgames.
```
Comment (rubenwardy) [inline downloads.html]: This is implied by above, not need to add this.

> You may also want to look at some <a href="https://forum.minetest.net/viewforum.php?f=15">subgames</a>

**Comment by treetrunk** [comment]
In the downloads page, I linked to the mods section of the customize page. Should it instead point to the forums like the Subgames link above it does?

**Comment by rubenwardy** [comment]
:+1:

I think to the customize page is good, as it provides a description of how mods etc work.

**Comment by rubenwardy** [comment]
Merged.

**Pull request closed without merging**